tests/neo.UnitTests/Trie/MPT/UT_MPTTrie.cs

```diff
@@ -0,0 +1,196 @@
+            
+
+            var r = new ShortNode();
+            r.Key = "0a0c".HexToBytes();
+
+
+            var b = new FullNode();
+            r.Next = b;
+
+            var l1 = new ShortNode();
+            l1.Key = new byte[]{0x01};
+            var v1 = new ValueNode();
+            v1.Value = "abcd".HexToBytes();
+            l1.Next = v1;
+            b.Children[0] = l1;
+
+            var l2 = new ShortNode();
+            l2.Key = new byte[]{0x09};
+            var v2 = new ValueNode();
+            v2.Value = "2222".HexToBytes();
+            l2.Next = v2;
+            b.Children[9] = l2;
+            
+            r1.Next = v1;
+            Assert.AreEqual("76248d1bf457f0b95c1f6d05d787dca152906f106bcbafacbf7a69c6ae1797c4", r1.GetHash().ToHexString());
+            Assert.AreEqual("f3ad94e8fb6e1e85a8b573b2343845e3b0e0b96b61fcd0e20b6df159fde137a7", r.GetHash().ToHexString());
+
+            var mpt = new MPTTrie(mptdb);
+            var result = true;
+            result = mpt.TryGet("0a0c0909".HexToBytes(), out byte[] value);
+            Assert.IsTrue(result);
+            result = mpt.TryDelete("0a0c0909".HexToBytes());
+            Assert.IsTrue(result);
+            result = mpt.TryDelete("0a0c0a0e".HexToBytes());
+            Assert.IsFalse(result);
+            Assert.AreEqual("76248d1bf457f0b95c1f6d05d787dca152906f106bcbafacbf7a69c6ae1797c4", mpt.GetRoot().ToHexString());
+        }
+
+        [TestMethod]
+        public void TestGetProof()
+        {
+            var r = new ShortNode();
+            r.Key = "0a0c".HexToBytes();
+            var b = new FullNode();
+            var l1 = new ShortNode();
+            l1.Key = new byte[]{0x01};
+            var l2 = new ShortNode();
+            l2.Key = new byte[]{0x09};
+            var v1 = new ValueNode();
+            v1.Value = "abcd".HexToBytes();
+            var v2 = new ValueNode();
+            v2.Value = "2222".HexToBytes();
+            var h1 = new HashNode();
+            h1.Hash = Encoding.ASCII.GetBytes("hello");
+            var l3 = new ShortNode();
+            l3.Next = h1;
+            l3.Key = "0e".HexToBytes();
+
+            r.Next = b;
+            b.Children[0] = l1;
+            l1.Next = v1;
+            b.Children[9] = l2;
+            l2.Next = v2;
+            b.Children[10] = l3;
+
+            var mpt = new MPTTrie(mptdb);
+
+            Assert.AreEqual("c32dc0dee8cec33436eff759ee460c65d1a22c0a65a5edd27c68dd80ac3963b4", mpt.GetRoot().ToHexString());
+            var dict = mpt.GetProof("0a0c0001".HexToBytes());
+            Assert.AreEqual(4, dict.Count);
+            Assert.IsTrue(dict.TryGetValue(mpt.GetRoot(), out byte[] value));
+            Assert.IsTrue(dict.TryGetValue(b.GetHash(), out value));
+            Assert.IsTrue(dict.TryGetValue(l1.GetHash(), out value));
+            Assert.IsTrue(dict.TryGetValue(v1.GetHash(), out value));                                                                        
+        }
+    }
+}
```

Comment:
A blank line should be added in the end of the file, otherwise the dotnet-format won't pass.